src/test/java/hudson/plugins/mercurial/FunctionalTest.java

```diff
@@ -197,6 +197,9 @@ private void assertClone(String log, boolean cloneExpected) {
         m.hg(repo, "branch", "b");
         m.touchAndCommit(repo, "b-1");
         FreeStyleProject p = j.createFreeStyleProject();
+        if (slave != null) {
+            p.setAssignedNode(slave);
+        }
```

Comment:
Just copied from the other test, `basics`: https://github.com/jenkinsci/mercurial-plugin/blob/08ac06536fb23f3edf2ef4a155ed6c495565eae8/src/test/java/hudson/plugins/mercurial/FunctionalTest.java#L169-L171